src/gui/lineedit.h

```diff
@@ -39,6 +42,11 @@ class LineEdit final : public QLineEdit
 public:
     explicit LineEdit(QWidget *parent = nullptr);
 
+signals:
```

Comment:
I don't insist, but I would use delayed notifying about "text changed" event as default behavior of our `LineEdit` class, i.e. call it `textChanged`.

Reply:
All comments addressed.